src/Controllers/CopyPatrol.php

```diff
@@ -164,7 +201,7 @@ protected function getRecords() {
 			}
 		}
 		// make this easier when working locally
-		$numRecords = $_SERVER['HTTP_HOST'] === 'localhost' ? 3 : 50;
+		$numRecords = $_SERVER['HTTP_HOST'] === 'localhost' ? 10 : 50;
```

Comment:
Not for this PR, but this limit should probably be a configuration option rather than relying on inspecting HTTP_HOST.